atlp-rwanda/tech-strikers-bn-frontend
# 175636784 setup codeclimate test coverage

#### What does this PR do?
setup codeclimate test coverage
#### Description of Task to be completed?
Connecting our github repo to codeclimate platform
Adding a test coverage badge in README.md file
Adding a maintainability badge in README.md file
#### How should this be manually tested?
run ``git clone https://github.com/atlp-rwanda/tech-strikers-bn-frontend.git``
run `cd tech-strikers-bn-frontend`
run `git checkout ch-setup-codeclimate-test-coverage-175636784`
Check for maintainability badge and test coverage badge in README.md file
#### Any background context you want to provide?
no
#### What are the relevant pivotal tracker stories?
[175636784](https://www.pivotaltracker.com/story/show/175636784)
